model/src/main/proto/question.proto

```diff
@@ -81,3 +81,17 @@ message AnsweredQuestionOutcome {
   // Whether the answer the learner submitted is the correct answer.
   bool is_correct_answer = 2;
 }
+
+// The final score and mastery calculations of the user after completing a training session.
+message QuestionAssessmentPerformance {
```

Comment:
Maybe include 'User' to emphasize that this is a user's performance within an assessment vs. the performance of the assessment itself.